docs/lake/design.md

```diff
@@ -140,27 +148,35 @@ and arbitrary data _shapes_ can coexist side by side.
 
 ### Query
 
-Data is read from a pool with the `query` command.
-A range can be specified with `-from` and/or `-to`.
-The default output format is ZNG though this can be overridden with `-f`
-to specify one of the various supported output formats.
+Data is read from one or more pools with the `query` command.  The pool names
+are specified with `from` at the beginning the Zed query along with an optional
+time range using `over` and `to`.  The default output format is ZNG though this
+can be overridden with `-f` to specify one of the various supported output
+formats.
 
-This example reads every record from the `logs` pool starting
-from time `2020-1-1T12:00` and sends the results as ZSON to stdout.
+This example reads every record from the full key range of the `logs` pool
+and sends the results as ZSON to stdout.
+
+```
+zed lake query -f zson 'from logs'
 ```
-zed lake query -p logs -f zson -from 2020-1-1T12:00
+
+Or we can narrow the span of the query by specifying the key range.
 ```
+zed lake query -z 'from logs over 2018-03-24T17:36:30.090766Z to 2018-03-24T17:36:30.090758Z'
```

Comment:
@mccanne: I'm effectively explaining this based on my black box testing and what I see in the ztests, so I'd appreciate your confirmation here that I'm not misrepresenting anything. From what I can tell, now that it's in the Zed, the key range:

1.  Must conform to a more strict format (i.e., the relaxed timestamp format like `2020-1-1T12:00` that previously worked when it was a `-from` parameter is now a syntax error if I try to use it inside the Zed)
2.  If I want to specify a key range in the Zed, it seems like I need to specify both the `over` _and_ `to`, whereas the prior example showed only the `-from`.

I'm not implying either of these are _problems_. But since they represent a change in functionality I just want to make sure they were intentional and not surprises.